Libraries build fail

Failed after https://github.com/dotnet/runtime/pull/62507

```sh
./build.sh --arch x64 --runtimeConfiguration Release --librariesConfiguration Release --subset clr+libs
```
output:
```
Build FAILED.

CSC : error CS2001: Source file '/home/z/Dev/runtime/src/libraries/Common/src/Interop/Interop.ZLib.cs' could not be found. [/home/z/Dev/runtime/src/libraries/System.IO.Compression/src/System.IO.Compression.csproj]
CSC : error CS2001: Source file '/home/z/Dev/runtime/src/libraries/Common/src/Interop/Interop.ZLib.cs' could not be found. [/home/z/Dev/runtime/src/libraries/System.Net.WebSockets/src/System.Net.WebSockets.csproj]
    0 Warning(s)
    2 Error(s)
```

Simple rename `src/libraries/Common/src/Interop/Interop.zlib.cs` -> `src/libraries/Common/src/Interop/Interop.ZLib.cs` needed?

--- a/src/libraries/System.IO.Compression/src/System.IO.Compression.csproj
+++ b/src/libraries/System.IO.Compression/src/System.IO.Compression.csproj
@@ -5,7 +5,7 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="$(CommonPath)Interop\Interop.ZLib.cs" />
+    <Compile Include="$(CommonPath)Interop\Interop.zlib.cs" />
     <Compile Include="$(SharedOpenSourcePath)System\IO\Compression\ZipArchive.cs" />
     <Compile Include="$(SharedOpenSourcePath)System\IO\Compression\ZipArchiveEntry.cs" />
     <Compile Include="$(SharedOpenSourcePath)System\IO\Compression\ZipArchiveMode.cs" />

--- a/src/libraries/System.Net.WebSockets/src/System.Net.WebSockets.csproj
+++ b/src/libraries/System.Net.WebSockets/src/System.Net.WebSockets.csproj
@@ -27,8 +27,8 @@
              Link="Common\System\IO\Compression\ZLibNative.cs" />
     <Compile Include="$(CommonPath)System\IO\Compression\ZLibNative.ZStream.cs"
              Link="Common\System\IO\Compression\ZLibNative.ZStream.cs" />
-    <Compile Include="$(CommonPath)Interop\Interop.ZLib.cs"
-             Link="Common\Interop\Interop.ZLib.cs" />
+    <Compile Include="$(CommonPath)Interop\Interop.zlib.cs"
+             Link="Common\Interop\Interop.zlib.cs" />
   </ItemGroup>
   <!-- Windows specific files -->
   <ItemGroup Condition=" '$(TargetsWindows)' == 'true'">

